### PR TITLE
add initial code

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

**Review by jasonrogena — DISMISSED** [review]

File: .github/worflows/publish-chart.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (jasonrogena) [inline .github/worflows/publish-chart.yml:32]:
```suggestion
          CR_TOKEN: "${{ secrets.GH_TOKEN }}"
```

Comment (jasonrogena) [reply .github/worflows/publish-chart.yml:32]: For some reason I can't create a secret called "GITHUB_TOKEN" but I can create "GH_TOKEN"

File: charts/opensrp-server-web/values.yaml [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: opensrp/opensrp-server-web
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```
Comment (jasonrogena) [inline charts/opensrp-server-web/values.yaml:11]:
```suggestion
  tag: "v2.5.11-SNAPSHOT"
```

File: charts/opensrp-server-web/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```
Comment (jasonrogena) [inline charts/opensrp-server-web/Chart.yaml:23]:
```suggestion
appVersion: v2.5.11-SNAPSHOT
```

**Review by morrismukiri — APPROVED** [review]

**Pull request merged**